cicecore/cicedynB/infrastructure/ice_restoring.F90

```diff
@@ -394,7 +394,11 @@ subroutine set_restore_var (nx_block, ny_block, &
             aicen(i,j,n) = c0
             vicen(i,j,n) = c0
             vsnon(i,j,n) = c0
-            trcrn(i,j,nt_Tsfc,n) = Tf(i,j)  ! surface temperature 
+            if (tmask(i,j)) then
+               trcrn(i,j,nt_Tsfc,n) = Tf(i,j)  ! surface temperature 
+            else
+               trcrn(i,j,nt_Tsfc,n) = c0  ! on land gridcells
+            endif
```

Comment:
Would it be more efficient (or maybe just more elegant) to use the merge function instead of all the if conditionals for setting Tsfc, here and elsewhere?

Reply:
I'm not a huge fan of merge syntax, but willing to start using it as it is a bit more compact.  Will keep that in mind moving forward, update the current implementation, and try to use merge more.